src/Models/HydrostaticFreeSurfaceModels/mg_implicit_free_surface_solver.jl

```diff
@@ -78,72 +96,17 @@ function MGImplicitFreeSurfaceSolver(grid::AbstractGrid,
     settings[:maxiter] = get(settings, :maxiter, grid.Nx * grid.Ny)
     settings[:reltol] = get(settings, :reltol, min(1e-7, 10 * sqrt(eps(eltype(grid)))))
 
-    right_hand_side = Field{Center, Center, Nothing}(grid)
-
     # initialize solver with Δt = nothing so that linear matrix is not computed;
     # see `initialize_matrix` methods
     solver = MultigridSolver(Az_∇h²ᶜᶜᶜ_linear_operation!, ∫ᶻ_Axᶠᶜᶜ, ∫ᶻ_Ayᶜᶠᶜ;
                              template_field = right_hand_side,
+                             matrix = matrix,
```

Comment:
```suggestion
                             matrix,
```

Cleaner.

After `;` just giving a variable implies that you are passing a keyword argument whose name is the variable's name...

```Julia
julia> a = 1; b = 2;

julia> (; a, b)
(a = 1, b = 2)

julia> (; a, b) == (a=a, b=b)
true
```